bokeh/models/images.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+from ..plot_object import PlotObject
+from ..properties import (Any, Dict, String, Int)
+
+class ImageSource(PlotObject):
+    """ A base class for all image source types. """
+
+    _args = ('url', 'extra_url_vars')
+
+    url = String("url", help="""
+    tile service url (example: http://c.tile.openstreetmap.org/{Z}/{X}/{Y}.png)
+    """)
+
+    extra_url_vars = Dict(String, Any(String, Int), help="""
```

Comment:
Wow dynamic typing sort of saves the day.

Yes `Dict(String, Any)` is what I want.  Changing now...